components/ContactCTA/index.tsx

```diff
@@ -1,16 +1,31 @@
 import Link from 'next/link'
+import { useLayoutEffect } from 'react'
 import { WebTranslations } from '../../i18n/web'
 import { useStore, useTranslation } from '../Hooks'
 
 export const ContactCTA: React.VFC = () => {
   const root = useStore()
   const tsln = useTranslation<WebTranslations>()
+
+  const handleClick = () => {
+    root.setActiveTab(2)
+  }
+
+  useLayoutEffect(() => {
+    const self = document.querySelectorAll('#faqLink a')[1]
+    if (!self) return
+
+    self.addEventListener('click', handleClick)
+    return () => self && self.removeEventListener('click', handleClick)
+  })
+
```

Comment:
need to `useLayoutEffect` because we want to synchronously add the event listener after the DOM. One annoyance of `dangerouslySetInnerHTML` is that I can't access it via normal DOM functions, so I had to add an id on the paragraph and query down to the anchor tag to add the listener.

It works now though!